src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -43,14 +43,16 @@ public class RequestExecutor : IDisposable
 
         private static readonly Guid GlobalApplicationIdentifier = Guid.NewGuid();
 
+        private static readonly TimeSpan MinHttpClientLifetime = TimeSpan.FromSeconds(10);
```

Comment:
Can we decrease that to 5s? This will give us up to only 20 http client instances within 100s TIME_WAIT period but would increase the robustness. Thoughts?

Reply:
done